src/FairPlaySocialSln/FairPlaySocial.Database/dbo/Tables/GroupMember.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE [dbo].[Group]
+(
+    [GroupId] BIGINT NOT NULL CONSTRAINT PK_Group_M PRIMARY KEY IDENTITY,
+    [MemberApplicationUserId]  BIGINT NOT NULL,
```

Comment:
Set a Foreign Key to [ApplicationUser].[ApplicationUserId]

Also, please create a unique index for the combination of columns
* GroupId
* MemberApplicationUserId

Reply:
Hey,
I have made required changes. Please let me know if any other modification required.